### ISSUE TITLE
"$HOME/.config/monitors.xml" generated by MATE is incompatible with GNOME and vice versa

### ISSUE THREAD

**Issue by davidhedlund** [mate-desktop/mate-control-center#712]
#### Expected behaviour
cinnamon is using a dedicated name, cinnamon-monitors.xml. This doesn't cause the file to be overwritten by the display preferences if other desktop environment are used on the same distro. If MATE and GNOME followed the same approach, the file name would be mate-monitors.xml, and gnome-monitors.xml, respectively.

#### Actual behaviour
GNOME's monitors.xml:

```
<monitors version="2">
  <configuration>
    <migrated/>
    <logicalmonitor>
      <x>0</x>
      <y>0</y>
      <primary>yes</primary>
      <monitor>
        <monitorspec>
          <connector>Virtual-1</connector>
          <vendor>???</vendor>
          <product>0x0000</product>
          <serial>0x00000000</serial>
        </monitorspec>
        <mode>
          <width>1600</width>
          <height>900</height>
          <rate>60</rate>
        </mode>
      </monitor>
    </logicalmonitor>
  </configuration>
  <configuration>
    <logicalmonitor>
      <x>0</x>
      <y>0</y>
      <scale>1</scale>
      <primary>yes</primary>
      <monitor>
        <monitorspec>
          <connector>Virtual-1</connector>
          <vendor>unknown</vendor>
          <product>unknown</product>
          <serial>unknown</serial>
        </monitorspec>
        <mode>
          <width>1600</width>
          <height>900</height>
          <rate>60</rate>
        </mode>
      </monitor>
    </logicalmonitor>
  </configuration>
</monitors>
```
MATE's monitors.xml:
```
<monitors version="1">
  <configuration>
      <clone>no</clone>
      <output name="Virtual-1">
          <vendor>???</vendor>
          <product>0x0000</product>
          <serial>0x00000000</serial>
          <width>1600</width>
          <height>900</height>
          <rate>60</rate>
          <x>0</x>
          <y>0</y>
          <rotation>normal</rotation>
          <reflect_x>no</reflect_x>
          <reflect_y>no</reflect_y>
          <primary>yes</primary>
      </output>
      <output name="Virtual-2">
      </output>
      <output name="Virtual-3">
      </output>
      <output name="Virtual-4">
      </output>
  </configuration>
</monitors>
```

#### Steps to reproduce the behaviour
Both GNOME and MATE can be installed on the same distro. Both DEs are installing "$HOME/.config/monitors.xml" via manually configured display GUI preferences:
* GNOME via `$ gnome-control-center display`
* MATE via `$ mate-display-properties`

The screen resolution is used directly (no need to log out). However, when you switch to the other desktop environment, the screen resolution is reset because their setup in monitors.xml is different.

I took this screenshot after I 1) Logged in to GNOMe, ran `$ gnome-control-center display` 2) logged out 3) logged into MATE
![Screenshot at 2023-07-23 20-34-08](https://github.com/mate-desktop/mate-control-center/assets/6416073/adffa08e-cf08-499d-8dc3-9e7800e4d7e9)

# MATE general version
1.26.0

# Package version
1.26.0

# Linux Distribution
Ubuntu MATE 22.04

#### Link to bugreport of your Distribution (requirement)
Will file it

**Comment by zhuyaliang** [comment]
We need to replace monitors.xml with mate-monitors.xml here
[monitors.xml](https://github.com/mate-desktop/mate-desktop/blob/master/libmate-desktop/mate-rr-config.c#L44)

**Comment by davidhedlund** [comment]
I filed the same issue at [gnome-control-center](https://gitlab.gnome.org/GNOME/gnome-control-center/-/issues/2579), but they moved it to [gnome-desktop](https://gitlab.gnome.org/GNOME/gnome-desktop/-/issues/224)

Should this issue be filed at [mate-desktop](https://github.com/mate-desktop/mate-desktop/issues) instead?

**Comment by davidhedlund** [comment]
> We need to replace `monitors.xml` with `mate-monitors.xml` here [monitors.xml](https://github.com/mate-desktop/mate-desktop/blob/master/libmate-desktop/mate-rr-config.c#L44)

Yes, the GNOME developers pointed out the same line for their project: https://gitlab.gnome.org/GNOME/gnome-control-center/-/issues/2579#note_1798271

**Comment by davidhedlund** [comment]
I just added this to the top post:
"Disclaimer: GNOME's monitors.xml is starting with <monitors version="2">, and that MATE's monitors.xml is starting with <monitors version="1">"
